cmd.h

```diff
@@ -29,6 +29,15 @@ class Dictionary {
 	void rmForce(string dirName, string currentFolder);
 	void touch(string fileName, string currentFolder);
 	void echo(string fileContent, string fileName, string currentFolder);
+	void mv(string currentFolder, string source, string dest);
+	bool searchFile(string fileName, string filePath);
+	bool searchDirectory(string forder, string subfolder);
+	string getLastElement(string fullPath);
+	string getPath(string fullPath, string currentFolder);
+	string getParentPath(string fullPath);
+	string convertRelativeToAbsolutePath(string relativePath, string currentFolder);
+	void moveFile(string sourcePath, string sourceLastElement, string destPath, string destLastElement);
+	void moveDirectory(string sourcePath, string sourceLastElement, string destPath, string destLastElement);
```

Comment:
Az ilyen segedfuggvenyeket erdemes lenne elrejteni a kulvilag elol. (nemcsak ezek.)

Reply:
@hegyhati 
1. "Legyen CI-ban teszt az uj funkciora" - ebben a branchben csak a mv command funkció lett megvalósítva, Dani dolgozik az első Googletest-es Unit testen, ha Ő elkészült, akkor egy újabb branchben nemcsak a mv, hanem az összes általam írt funkcióhoz (echo, touch, mv) tartozóUnit testet be fogom pótolni.
2. path kinyerős témához: a main függvényben található path kezelés (52-72), a split függvény (14-24) egyszerre valósít meg input paraméter feldolgozást, aktuális mappába lépést és az ehhez tartozó relatív path kezelést, ezért az általam írt relatív útvonalas függvények nem integrálhatóak.
3.segédfüggvényeket elrejtettem, a változtatások pusholva egy kommitban.

Reply:
Tyuuu, erdemes volna mindent oda kommentelni, ahol fel lett vetve a kerdes, igy kicsit kaotikus. 1-re reagalva: nem a unit testekre gondoltam, hanem arra a tesztre, amit mar elso heten csinaltatok, hogy rakulditek az inputot, es nezitek mi lesz. hogy abban legyen hasznalva az mv is.